core-bundle/assets/styles/component/choices.pcss

```diff
@@ -0,0 +1,200 @@
+@import "choices.js/public/assets/styles/choices.css";
+
+.choices {
+    position: relative;
+    overflow: hidden;
+    display: inline-block;
+    padding: 0 !important;
+    font-size: .875rem;
+    vertical-align: middle;
+    width: 100%;
+    height: 30px;
+    box-sizing: border-box;
+    border: 1px solid var(--form-border);
+    border-radius: 2px;
+    background: var(--form-bg) url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMiIgaGVpZ2h0PSIxMiIgZmlsbD0ibm9uZSIgc3Ryb2tlPSIjNmE2YTZjIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIHN0cm9rZS13aWR0aD0iMS41IiB2aWV3Qm94PSIwIDAgMjQgMjQiPjxwYXRoIGQ9Im02IDkgNiA2IDYtNiIvPjwvc3ZnPgo=") right -16px top 3px no-repeat;
+    background-origin: content-box;
+    cursor: pointer;
+    text-align: left;
```

Comment:
```suggestion
    text-align: left;
        margin: 3px 0;

    /* Add the tl_select margin since the styles are applied in a a shadow dom */
    &:last-child {
        margin-bottom: 3px;
    }
```

Just this margin here:
![image](https://github.com/user-attachments/assets/4d2ef79a-6afd-4254-9ece-d40fe851f89b)

Reply:
See f907a1d5280c85c044cfb02e2bb1808e613700fa